server/db/seeds/students_assessments.js

```diff
@@ -0,0 +1,13 @@
+exports.seed = function (knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('students_assessments').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('students_assessments').insert([
+        {id: 1, student_id: 1, assessment_id: 1, status_id: 4, evidence: 'evidence' , date_modified: 'date'},
+        {id: 1, student_id: 1, assessment_id: 2, status_id: 3, evidence: 'evidence', date_modified: 'date'},
+        {id: 1, student_id: 1, assessment_id: 3, status_id: 2, evidence: 'evidence', date_modified: 'date'},
+        {id: 1, student_id: 1, assessment_id: 4, status_id: 1, evidence: 'evidence', date_modified: 'date'}
```

Comment:
I've just merged a PR that changes the data type of `data_modified` to a _datetime_. Please update the seed data to reflect this :-)
You may want to use [Date.now()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now)